test/ses-compat.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+// requiring ses exposes "lockdown" on the global
+require('ses');
```

Comment:
that's a strange pattern, why not just export `lockdown`?

Reply:
agreed :shrug: